unit-tests/motion/test-motion.py

```diff
@@ -0,0 +1,58 @@
+# License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Lets place this UT under the existing `sw-dev` folder.
Then the name will be
test-sw-dev-motion

Reply:
done